molgenis-core-ui/src/main/javascript/modules/react-components/AlertMessage.js

```diff
@@ -1,14 +1,14 @@
-/* global _: false, React: false, molgenis: true */
-(function(_, React, molgenis) {
-	"use strict";
-	
+import React from "react";
+import DeepPureRenderMixin from "./mixin/DeepPureRenderMixin";
+import Icon from "./Icon";
+
 	var div = React.DOM.div, span = React.DOM.span, button = React.DOM.button;
```

Comment:
Yes. Except then git no longer recognizes the updates as file moves and changes at the top and bottom. It simply sees them as deletions and creations. Then it's hard to judge the pull request.